pages/community/code-of-conduct.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: page
+title: Code of Conduct
+permalink: /code-of-conduct/
+subtitle: The US-RSE Association Code of Conduct.
+---
+
+# US-RSE Association Diversity Statement
+US-RSE Association welcomes and encourages participation in our community by people of all backgrounds and identities. We, the members
+of the US-RSE Association, are committed to promoting and sustaining a culture that values mutual respect, tolerance, and learning, and we work together as a community to help each other live out these values.
+
+We have created this diversity statement because we believe that a diverse community is stronger, more vibrant, and produces better
+software and better research. A diverse community where people treat each other with respect has more potential contributors,
+more sources for ideas, and fewer shared assumptions that might hinder the achievement of our goals.
+
+Although we have phrased the formal diversity statement generically to make it all-inclusive, we recognize that there are specific
+identities that are impacted by systemic discrimination and marginalization. We welcome all people to participate in the US-RSE
+association regardless of their identity or background.
+
+## US-RSE Association Code of Conduct: Introduction & Scope
```

Comment:
Change to 
```
## Introduction and Scope
```

Reply:
+1 this would fix the first bit of comments that I made above!